woof-code/rootfs-skeleton/usr/local/petget/installpkg.sh

```diff
@@ -244,6 +245,15 @@ elif [ $PUPMODE -eq 13 ];then
 	fi
 fi
 
+
+if [ -L /bin ] || [ -L /lib ] || [ -L /lib64 ] || [ -L /lib32 ] || [ -L /libx32 ] || [ -L /sbin ]; then
```

Comment:
I though 3 tests were too many to identify a usr-merged distro. 6 is way an overkill
/bin and or /lib should be enough

Reply:
It was patterned on Ubuntu usrmerge. Its better to anticipate the packages' structure rather than creating trouble or simply called future proofing